Detectors/FIT/workflow/include/FITWorkflow/RawReaderFIT.h

```diff
@@ -151,6 +161,7 @@ class RawReaderFIT : public RawReaderType
     if constexpr (sUseTrgInput) {
       outputSpec.emplace_back(mDataOrigin, DetTrigInput_t::sChannelNameDPL, 0, o2::framework::Lifetime::Timeframe);
     }
+    outputSpec.emplace_back(mDataOrigin, "RawDataMetric", 0, o2::framework::Lifetime::Timeframe);
```

Comment:
should the RawDataMetrics not be sent only when requested and not unconditionally?

Reply:
@martenole shall I make raw data metric sending optional for DPL task? Actually I forgot, in case if it sends messages with `Lifetime::Timeframe` should there be always receiver task? Raw data metric objects will be used only for QC

Reply:
I think I misread. You want to have only the dumping of raw data metrics optionally, but they should always be produced. I think that is OK. If there is no consumer the message should just be dropped by the framework.

Reply:
Okay, thanks! Dumping is just for debugging